#scan_topic: /base_laser_rear/scan
scan_topic: /scan_unified
transform_link: base_link



isOnePersonToTrack: false

# front_cutted.bag
x_lower_limit: -2.6
x_upper_limit: 1.0
y_lower_limit: -1.0
y_upper_limit: 2.8

# four.bag
#x_lower_limit: -2.2
#x_upper_limit: 0.4
#y_lower_limit: -1.9
#y_upper_limit: 1.8


#x_lower_limit: -2.2
#x_upper_limit: 0.6
#y_lower_limit: -2.3
#y_upper_limit: 0.8

# one person
#x_lower_limit: -1.9
#x_upper_limit: -0.2
#y_lower_limit: -0.8
#y_upper_limit: 0.8

leg_radius: 0.1
min_observations: 3
min_predictions: 5
max_dist_btw_legs: 0.8
z_coordinate: 0.0
vel_stance_threshold: 0.47
vel_swing_threshold: 0.93
state_dimensions: 6
clusterTolerance: 0.09
minClusterSize: 3
maxClusterSize: 100
max_nn_gating_distance: 1.0
occluded_dead_age: 5
variance_observation: 0.25
min_dist_travelled: 0.25
max_cov: 0.81
#history_size: 5
